Resolve Windows testing error

xref #4 
```py
2020-06-27T17:42:19.0854347Z (%PREFIX%) %SRC_DIR%>cd tests   && %PREFIX%\python.exe runAll.py 
2020-06-27T17:42:53.4126366Z ..................................................................................s..................................................s......s..s................................................................................tempfile.tif: Cannot read TIFF header.
2020-06-27T17:42:53.4296355Z Etempfile.tif: Cannot read TIFF header.
2020-06-27T17:43:05.5066350Z E..............................................sss..s.....s................................................................................
2020-06-27T17:43:05.5067038Z ======================================================================
2020-06-27T17:43:05.5067371Z ERROR: test0 (test_pdfgen_callback.CallBackTestCase)
2020-06-27T17:43:05.5067737Z Make a PDFgen document with most graphics features
2020-06-27T17:43:05.5068045Z ----------------------------------------------------------------------
2020-06-27T17:43:05.5068390Z Traceback (most recent call last):
2020-06-27T17:43:05.5068786Z   File "D:\bld\reportlab_1593279401970\test_tmp\tests\test_pdfgen_callback.py", line 24, in test0
2020-06-27T17:43:05.5069206Z     makeDocument(outputfile('test_pdfgen_callback.pdf'), pageCallBack=self.callMe)
2020-06-27T17:43:05.5069672Z   File "D:\bld\reportlab_1593279401970\test_tmp\tests\test_pdfgen_general.py", line 738, in makeDocument
2020-06-27T17:43:05.5070043Z     c.drawInlineImage(tif,1*inch, 1*inch)
2020-06-27T17:43:05.5070480Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\canvas.py", line 859, in drawInlineImage
2020-06-27T17:43:05.5070892Z     img_obj = PDFImage(image, x,y, width, height)
2020-06-27T17:43:05.5071336Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 40, in __init__
2020-06-27T17:43:05.5071900Z     self.getImageData()
2020-06-27T17:43:05.5072313Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 158, in getImageData
2020-06-27T17:43:05.5072744Z     imagedata, imgwidth, imgheight = self.non_jpg_imagedata(image)
2020-06-27T17:43:05.5073231Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 138, in non_jpg_imagedata
2020-06-27T17:43:05.5073660Z     imagedata = pdfutils.cacheImageFile(image,returnInMemory=1)
2020-06-27T17:43:05.5074140Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfbase\pdfutils.py", line 95, in cacheImageFile
2020-06-27T17:43:05.5074562Z     code = makeA85Image(filename,IMG)
2020-06-27T17:43:05.5074947Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfbase\pdfutils.py", line 38, in makeA85Image
2020-06-27T17:43:05.5075339Z     raw = img.getRGBData()
2020-06-27T17:43:05.5075703Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 893, in getRGBData
2020-06-27T17:43:05.5076136Z     annotateException('\nidentity=%s'%self.identity())
2020-06-27T17:43:05.5076554Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 1397, in annotateException
2020-06-27T17:43:05.5077012Z     rl_reraise(t,v,b)
2020-06-27T17:43:05.5077358Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 150, in rl_reraise
2020-06-27T17:43:05.5077723Z     raise v
2020-06-27T17:43:05.5078058Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 888, in getRGBData
2020-06-27T17:43:05.5078446Z     im = im.convert('RGB')
2020-06-27T17:43:05.5078784Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\Image.py", line 901, in convert
2020-06-27T17:43:05.5079129Z     self.load()
2020-06-27T17:43:05.5079465Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\TiffImagePlugin.py", line 1070, in load
2020-06-27T17:43:05.5079855Z     return self._load_libtiff()
2020-06-27T17:43:05.5080266Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\TiffImagePlugin.py", line 1182, in _load_libtiff
2020-06-27T17:43:05.5080621Z     raise OSError(err)
2020-06-27T17:43:05.5081088Z OSError: (-2, "\nidentity=[ImageReader@0x26e8d3695c0 filename='D:\\\\bld\\\\reportlab_1593279401970\\\\test_tmp\\\\tests\\\\test-cross.tiff']")
2020-06-27T17:43:05.5081450Z 
2020-06-27T17:43:05.5081820Z ======================================================================
2020-06-27T17:43:05.5082136Z ERROR: test0 (test_pdfgen_general.PdfgenTestCase)
2020-06-27T17:43:05.5082455Z Make a PDFgen document with most graphics features
2020-06-27T17:43:05.5082766Z ----------------------------------------------------------------------
2020-06-27T17:43:05.5083112Z Traceback (most recent call last):
2020-06-27T17:43:05.5083494Z   File "D:\bld\reportlab_1593279401970\test_tmp\tests\test_pdfgen_general.py", line 1044, in test0
2020-06-27T17:43:05.5083890Z     run(outputfile('test_pdfgen_general.pdf'))
2020-06-27T17:43:05.5084247Z   File "D:\bld\reportlab_1593279401970\test_tmp\tests\test_pdfgen_general.py", line 1005, in run
2020-06-27T17:43:05.5084610Z     c = makeDocument(filename)
2020-06-27T17:43:05.5084959Z   File "D:\bld\reportlab_1593279401970\test_tmp\tests\test_pdfgen_general.py", line 738, in makeDocument
2020-06-27T17:43:05.5085355Z     c.drawInlineImage(tif,1*inch, 1*inch)
2020-06-27T17:43:05.5085754Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\canvas.py", line 859, in drawInlineImage
2020-06-27T17:43:05.5086192Z     img_obj = PDFImage(image, x,y, width, height)
2020-06-27T17:43:05.5086643Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 40, in __init__
2020-06-27T17:43:05.5087085Z     self.getImageData()
2020-06-27T17:43:05.5087451Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 158, in getImageData
2020-06-27T17:43:05.5087914Z     imagedata, imgwidth, imgheight = self.non_jpg_imagedata(image)
2020-06-27T17:43:05.5088393Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfgen\pdfimages.py", line 138, in non_jpg_imagedata
2020-06-27T17:43:05.5088823Z     imagedata = pdfutils.cacheImageFile(image,returnInMemory=1)
2020-06-27T17:43:05.5089290Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfbase\pdfutils.py", line 95, in cacheImageFile
2020-06-27T17:43:05.5089677Z     code = makeA85Image(filename,IMG)
2020-06-27T17:43:05.5090109Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\pdfbase\pdfutils.py", line 38, in makeA85Image
2020-06-27T17:43:05.5090468Z     raw = img.getRGBData()
2020-06-27T17:43:05.5090865Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 893, in getRGBData
2020-06-27T17:43:05.5091260Z     annotateException('\nidentity=%s'%self.identity())
2020-06-27T17:43:05.5091716Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 1397, in annotateException
2020-06-27T17:43:05.5092068Z     rl_reraise(t,v,b)
2020-06-27T17:43:05.5092454Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 150, in rl_reraise
2020-06-27T17:43:05.5092818Z     raise v
2020-06-27T17:43:05.5093147Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\reportlab\lib\utils.py", line 888, in getRGBData
2020-06-27T17:43:05.5093533Z     im = im.convert('RGB')
2020-06-27T17:43:05.5093874Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\Image.py", line 901, in convert
2020-06-27T17:43:05.5094221Z     self.load()
2020-06-27T17:43:05.5094558Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\TiffImagePlugin.py", line 1070, in load
2020-06-27T17:43:05.5094946Z     return self._load_libtiff()
2020-06-27T17:43:05.5095319Z   File "D:\bld\reportlab_1593279401970\_test_env\lib\site-packages\PIL\TiffImagePlugin.py", line 1182, in _load_libtiff
2020-06-27T17:43:05.5095703Z     raise OSError(err)
2020-06-27T17:43:05.5096131Z OSError: (-2, "\nidentity=[ImageReader@0x26e8c261160 filename='D:\\\\bld\\\\reportlab_1593279401970\\\\test_tmp\\\\tests\\\\test-cross.tiff']")
2020-06-27T17:43:05.5096521Z 
2020-06-27T17:43:05.5096813Z ----------------------------------------------------------------------
2020-06-27T17:43:05.5097721Z Ran 364 tests in 45.802s
2020-06-27T17:43:05.5097863Z 
2020-06-27T17:43:05.5098027Z FAILED (errors=2, skipped=9)
2020-06-27T17:43:05.6260951Z 
2020-06-27T17:43:05.6261733Z (%PREFIX%) %SRC_DIR%\tests>IF 1 NEQ 0 exit /B 1 
2020-06-27T17:43:05.6268096Z 
2020-06-27T17:43:05.6268576Z (%PREFIX%) %SRC_DIR%\tests>IF 1 NEQ 0 exit 1 
2020-06-27T17:43:07.0978553Z Tests failed for reportlab-3.5.44-py36h4243904_0.tar.bz2 - moving package to D:\bld\broken
2020-06-27T17:43:07.0979610Z WARNING:conda_build.build:Tests failed for reportlab-3.5.44-py36h4243904_0.tar.bz2 - moving package to D:\bld\broken
2020-06-27T17:43:07.0980411Z WARNING conda_build.build:tests_failed(2751): Tests failed for reportlab-3.5.44-py36h4243904_0.tar.bz2 - moving package to D:\bld\broken
2020-06-27T17:43:07.1486787Z TESTS FAILED: reportlab-3.5.44-py36h4243904_0.tar.bz2
2020-06-27T17:43:07.7514905Z ##[error]Cmd.exe exited with code '1'.
2020-06-27T17:43:07.7940241Z ##[section]Finishing: Build recip
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -20,9 +20,9 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
+    - pip
     - freetype
     - pillow >=2.4.0
-    - pip
   run:
     - python
     - freetype
@@ -40,13 +40,6 @@ test:
     - reportlab.pdfbase
     - reportlab.pdfgen
     - reportlab.platypus
-  source_files:
-    - docs/
-    - tests/
-    - tools/
-  commands:
-    - cd tests && $PYTHON runAll.py    # [not win]
-    - "cd tests && %PYTHON% runAll.py" # [win]
 
 about:
   home: https://www.reportlab.com/opensource/

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -40,15 +40,13 @@ test:
     - reportlab.pdfbase
     - reportlab.pdfgen
     - reportlab.platypus
-  source_files:  # [not win]
-    - docs/      # [not win]
-    - tests/     # [not win]
-    - tools/     # [not win]
+  source_files:
+    - docs/
+    - tests/
+    - tools/
   commands:
     - cd tests && $PYTHON runAll.py    # [not win]
-    # Skip full Windows tests for now. See
-    # https://github.com/nsls-ii-forge/reportlab-feedstock/pull/4 for details.
-    # - "cd tests && %PYTHON% runAll.py"  # [win]
+    - "cd tests && %PYTHON% runAll.py" # [win]
 
 about:
   home: https://www.reportlab.com/opensource/

